pulsar-broker/src/test/java/org/apache/pulsar/broker/admin/NamespacesV2Test.java

```diff
@@ -53,7 +53,7 @@
 import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.Test;
 
-@Test(groups = "broker-admin-v2")
+@Test(groups = "broker-admin")
```

Comment:
Why change this line?

Reply:
> Why change this line?

In https://github.com/apache/pulsar/blob/master/build/run_unit_group.sh, there is no "broker-admin-v2"

Reply:
Got it, thanks!